src/main/java/org/gridsuite/cases/importer/job/CaseImportServiceRequester.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.cases.importer.job;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.Duration;
+import java.util.*;
+
+/**
+ * @author Nicolas Noir <nicolas.noir at rte-france.com>
+ */
+public class CaseImportServiceRequester {
+
+    private static final String API_VERSION = "v1";
+
+    private static String serviceUrl;
+
+    private final HttpClient httpClient;
+
+    public CaseImportServiceRequester(String serviceUrl) {
+        this.serviceUrl = serviceUrl;
+        httpClient = HttpClient.newHttpClient();
```

Comment:
Are you sure it is not needed to close it?

Reply:
Actually, it seems HttpClients do not need to be closed